mainapp/models.py

```diff
@@ -184,3 +182,43 @@ class DistrictCollection(models.Model):
     collection = models.TextField(
         verbose_name="Details of collected items"
     )
+
+class RescueCamp(models.Model):
+    verbose_name = 'Relief Camp'
+    name = models.CharField(max_length=50,verbose_name="Name")
+    location = models.TextField(verbose_name="Address",blank=True,null=True)
+    district = models.CharField(
+        max_length=15,
+        choices=districts
+    )
+    contacts = models.TextField(verbose_name="Phone Numbers",blank=True,null=True)
+    data_entry_user = models.ForeignKey(User,models.SET_NULL,blank=True,null=True)
+    class Meta:
+        verbose_name = 'Relief Camp'   
+    def __str__(self):
+        return self.name
+
+
+class Person(models.Model):
+    name = models.CharField(max_length=30,blank=False,null=False,verbose_name="Name - പേര്")
+    phone = models.CharField(max_length=11,null=True,blank=True,verbose_name='Mobile - മൊബൈൽ')
+    age = models.IntegerField(null=True,blank=True,verbose_name="Age - പ്രായം")
+    gender = models.IntegerField(
+        choices = gender,
+        verbose_name='Gender - ലിംഗം',
+        null=True,blank=True
+    )
+    address = models.CharField(max_length=150,null=True,blank=True,verbose_name="Address - വിലാസം")
+    district = models.CharField(
+        max_length = 15,
+        choices = districts,
+        verbose_name='District - ജില്ല',
+        null=True,blank=True
+    )
+    notes = models.TextField(max_length=500,null=True,blank=True,verbose_name='Notes - കുറിപ്പുകൾ')
+    camped_at = models.ForeignKey(RescueCamp,models.CASCADE,blank=False,null=False,verbose_name='Camp Name - ക്യാമ്പിന്റെ പേര്')
+    added_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Suggestion
Add `is_active` Boolean field - for volunteers to clear names as people get reunited with family.
Also, make sure that we filter our `is_active=False` from query results and the ability for volunteers to edit the status of a person.
This requires edit feature - which can be added later tomorrow after we roll out the first version without edit in UI, but `is_active` in models